modules/just/just.nu

```diff
@@ -0,0 +1,68 @@
+def "nu-complete just recipes" [] {
```

Comment:
why are these completions in the module directory?

Reply:
because they are related. The same is true for other modules, such as kuberentes.nu, which also has many completions. It would be absurd to ask for completions to be separated in this scenario.

The current `just.nu` can also support npm at the same time, and can also support others, if you want.

I think this form is reasonable now, if you don't think so, close this pr.

Reply:
I really don't want to have mutliple just.nu completions spread throughout the repo. I'd prefer the just completions be merged with the other ones. I'm fine with the git and kub ones.

Reply:
Ok, I will remove `just.nu`, but without adding other taskrunner support, or allowing plugin extensions, I can only use my own version and will not update it.

By the way, I don't think I will use taskrunner other than just, but tasks of vscode can be considered to support it. However, vscode tasks does not have a standalone runner, so I don't know if it is worth implementing another runner.

On the other hand, nushell itself is very suitable to be a taskrunner. But there seems to be no way to complete the parameters of the main function of the script?

Reply:
I'm not sure